src/components/KeyboardAwareScrollView/index.tsx

```diff
@@ -26,6 +26,7 @@ type KeyboardAwareScrollViewProps = {
   bottomOffset?: number;
   /** Prevents automatic scrolling of the `ScrollView` when the keyboard gets hidden, maintaining the current screen position. Default is `false`. */
   disableScrollOnKeyboardHide?: boolean;
+  enableKeyboardHandling?: boolean;
```

Comment:
Can you also write JSDoc for this field?

And can you also mention this filed in the documentation (1.10.0 + latest version)?